src/content/en/2021/cms.md

```diff
@@ -49,76 +46,78 @@ There is great variability in the type and scope of the support CMSs provide for
+  description="Column chart showing the adoption of CMSs over the past 3 years. 45% of desktop websites and 46% of mobile websites are built using a CMS in 2021. For 2020, both desktop and mobile are 42%, while in 2019 they are both 40%.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRzYkQYaBTpD5FKloDXsdb32Y2pjdxFZq_LoekECgTy53F-dRATm9wFFJ3dDVDS8_cGn0h2mKDBjgdM/pubchart?oid=551554892&format=interactive",
   sheets_gid="362035103",
   sql_file="cms_adoption.sql"
 ) }}
 
-As of July 2021, over 45% of public websites are powered by a CMS platform, indicating growth of over 7% from 2020. This breaks down to 45% on desktop, up from 42% in 2019, and 46% on mobile, up from 42% in 2020.
+As of July 2021, over 45% of public websites are powered by a CMS platform, indicating growth of over 7% [from 2020](../2020/cms#cms-adoption). This breaks down to 45% on desktop, up from 42% in 2019, and 46% on mobile, up from 42% in 2020.
+
+It is interesting to compare these numbers with another commonly used dataset, such as <a hreflang="en" href="https://w3techs.com/technologies/history_overview/content_management/all/q">W3Techs</a>, which reported that as of July 2021, 64.6% of websites are created using a CMS, up from 59.2% in July 2020, which is an increase of over 9%.
 
-It is interesting to compare these numbers with another commonly used dataset, such as [W3Techs](https://w3techs.com/technologies/history_overview/content_management/all/q), which reported that as of July 2021, 64.6% of websites are created using a CMS, up from 59.2% in July 2020, which is an increase of over 9%.
+The deviation between our analysis and W3Techs' analysis can be explained by a difference in research methodologies, and the definition of what is a CMS.
 
-The deviation between our analysis and W3Techs’ analysis can be explained by a difference in research methodologies, and the definition of what is a CMS.
+W3Techs definition is the following: "_Content Management Systems are applications for creating and managing the content of a website. We include all such systems in this category, also systems that are often classified as wikis, blog engines, discussion boards, static site generators, website editors or any type of software that provides website content_."
 
-W3Techs definition is the following: "_Content management systems are applications for creating and managing the content of a website. We include all such systems in this category, also systems that are often classified as wikis, blog engines, discussion boards, static site generators, website editors or any type of software that provides website content_."
+{# TODO You don't explain whats different? I guess it's implied that Wappalyzer has a stricter definition excluding Shopify and the like, but should be be more explicit on this rather than assuming the readers will follow? #}
```

Comment:
@alonkochba thoughts on this?

Reply:
Added another sentence, let me know if it's better.
I already expand on this in the "What is a CMS" section, so it felt implied, but now it's a bit more explicit.

Reply:
Thanks. Agree it's basically covered about but think it's better reiterating. Suggested a very slight tweak but resolving this.